terraform/cloudwatch_dashboard.template.json.py

```diff
@@ -49,10 +50,14 @@ def dashboard_body(name: str):
                     '''),
                     'region': config.region,
                     'stacked': True,
-                    'title': 'Aggregation outcomes in # of contributions',
+                    'title': (
```

Comment:
The query was not updated to match title, so this will still show aggregation outcomes on the mirror dashboard.